File: app/src/main/java/com/example/recreate/java/student.java
package com.example.recreate.java;

public class student {
    String name;
    int age;
    String pa;

    public student(String name, int age, String pa){
        this.name=name;
        this.age=age;
        this.pa=pa;
    }

    public String getName(){
        return name;
    }

    public void setName(){
        this.name=name;
    }

    public int getAge(){
        return age;
    }

    public void setAge(){
        this.age=age;
    }

    public String getPa(){
        return pa;
    }

    public void setPa(){
        this.pa=pa;
    }
}

File: app/src/main/java/com/example/recreate/kotlin/Student.kt
package com.example.recreate.kotlin

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "student_table")
data class Student(
    @PrimaryKey(autoGenerate = true) val id: Int = 0,
    val name: String,
    val age: Int,
    val grade: String
)
